docs/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+## Changes Made:
```

Comment:
It would be so cool if you could get this to pop up in our commit message prompt